.github/workflows/tests.yml

```diff
@@ -82,7 +82,7 @@ jobs:
 
       - name: Run tests
         run: |
-          pytest tests/ --deselect=tests/notebooks -n 8 -x
+          pytest tests/ --deselect=tests/notebooks -n 8 -x --durations=50
```

Comment:
Q: Why this? (sometimes the tests hangs and we should stop them?)

Reply:
I included this to list the top 50 slowest tests, so we could ,check the efficiency of the changes added in the PR, if it drastically slow, we could request to optimize it

Reply:
It willl not repeat the tests george.I thought to keep it as a benchmark to prevent from slowing the CI